### ISSUE TITLE
why compute bernoulli entropy in this way?

### ISSUE THREAD

**Issue by lucas-yang256** [openai/imitation#13]
as the code written [here](def logit_bernoulli_entropy(logits_B):
    ent_B = (1.-tensor.nnet.sigmoid(logits_B))*logits_B - logsigmoid(logits_B)
    return ent_B), bernoulli was computed by [this way](https://github.com/openai/imitation/blob/99fbccf3e060b6e6c739bdf209758620fcdefd3c/policyopt/thutil.py#L48-L51)
https://github.com/openai/imitation/blob/99fbccf3e060b6e6c739bdf209758620fcdefd3c/policyopt/thutil.py#L48-L51

but it's different to the equation of binary entropy:
$-p\log p - (1-p)\log(1-p)$

is there any relationship between these two expressions? or why does openai compute bernoulli entropy that way? is there any theoretical equation support?
